addons/web/static/src/js/views/kanban/kanban_column_progressbar.js

```diff
@@ -80,28 +80,32 @@ var KanbanColumnProgressBar = Widget.extend({
             // This should be executed when the progressbar is fully rendered
             // and is in the DOM, this happens to be always the case with
             // current use of progressbars
-
-            var subgroupCounts = {};
-            let allSubgroupCount = 0;
-            _.each(self.colors, function (val, key) {
-                var subgroupCount = self.columnState.progressBarValues.counts[key] || 0;
-                if (self.activeFilter === key && subgroupCount === 0) {
-                    self.activeFilter = false;
-                }
-                subgroupCounts[key] = subgroupCount;
-                allSubgroupCount += subgroupCount;
-            });
-            subgroupCounts.__false = self.columnState.count - allSubgroupCount;
-
-            self.groupCount = self.columnState.count;
-            self.subgroupCounts = subgroupCounts;
-            self.prevTotalCounterValue = self.totalCounterValue;
-            self.totalCounterValue = self.sumField ? (self.columnState.aggregateValues[self.sumField] || 0) : self.columnState.count;
-
+            self.resetCounters();
             self._notifyState();
             self._render();
         });
     },
+    /**
+     * Computes the count of each sub groups and the total count
```

Comment:
each sub group

Reply:
reset is probably not a good name then (according to the docstring). I'd suggest 'computeCounters' or 'updateCounters'